### DISCUSSION TITLE
[인프라] 도커 네트워크

### DISCUSSION THREAD

**Discussion by pipisebastian** [BDD-CLUB/challenge-every-week-posting#124] [1. 개발 관련 글 모음]
호호.. 반성문 2개 써야해용...

https://velog.io/@pipi/docker-%EB%8F%84%EC%BB%A4-%EB%84%A4%ED%8A%B8%EC%9B%8C%ED%81%AC

**Comment by llddang** [comment]
아하 저런식으로 docker 사용할때 ip가 설정되는 구나!
그림 있어서 더 알아보기 쉬웠당!!!
